modules/jooby-pac4j/src/main/java/io/jooby/pac4j/Pac4jModule.java

```diff
@@ -581,7 +581,7 @@ public void install(@NonNull Jooby application) throws Exception {
           });
     }
     /** Set current user provider */
-    application.setCurrentUser(new Pac4jCurrentUser());
+    application.setCurrentUser(new Pac4jCurrentUser(pac4j));
```

Comment:
Let's make the pac4j configuration accesible by adding to registry:

```
application.getServices().put(Config.class, pac4j);
```

Reply:
OK. I added the configuration to the registry at the end of the ``install`` method because above, the configuration can be changed